docs/UserGuide.adoc

```diff
@@ -49,287 +49,289 @@ This app will not work with earlier versions of Java 8.
 
-* **`+issue`**`ti/TestIssue` : creates an issue on github in the team repo
+=== Overview
 
-* *`nl`* : adds the default TODO list
+Now you come to use the ProgressChecker. You can type the command in the command box and press kbd:[Enter] to execute it. Here are the features that you can try:
 
-* *`vt`* : views the TODO list
+[TIP]
+You can type a command and press `Tab` to auto bring out all the command parameters.
 
-* **`ct`**`3` : mark task No.3 as completed
+* *`help`* : you can open the help window in case you have any questions.
 
-* **`rt`**`3` : reset task No.3 as incompleted
+* *`list`* : you can see all contacts will be listed on the left side of the window.
 
-* **`ans`**`11.1.1 sample answer` : answer exercise 11.1.1 with `sample answer`
+* *`add n/John Doe p/98765432 e/johnd@example.com m/Computer Science y/2`* : you can add the contact indormation of your friend `John Doe` to the ProgressChecker. He is in his 2nd year majoring in computer engineering.
 
-* *`exit`* : exits the app
+* *`+issue ti/TestIssue`* : you can create an issue on github in your team repo.
 
-.  Refer to <<Features,Features>> for details of each command.
+* *`nl`* : you can create the default TODO list for your the tasks to complete this week.
 
-[[Features]]
-== Features
+* *`vt`* : you can view the tasks in the TODO list.
+
+* *`ct 3`* : you can mark task No.3 as completed when you finish it.
+
+* *`rt 3`* : you can reset task No.3 as incompleted if you mark it completed by mistake or you want to update more details.
 
-You may find having to learn a large new number of commands rather intimidating. The section below specifies all the commands in details with examples to help you overcome that learning curve and let you make the best out of ProgressChecker.
+* *`ans 11.1.1 your answer`* : you can answer exercise 11.1.1 with `your answer`.
 
+* *`exit`* : you can exit the app.
+
+Refer to <<Default,below>> for details of each command.
 ====
-*Command Format*
+*Notes:*
 
 * The commands are case-insensitive. However, for simplicity all the examples have the commands in lower case.
 * You can auto-complete any command by pressing tab key.
 * Words in `UPPER_CASE` are the parameters to be supplied by the user e.g. in `add n/NAME`, `NAME` is a parameter which can be used as `add n/John Doe`.
 * Items in square brackets are optional e.g `n/NAME [t/TAG]` can be used as `n/John Doe t/friend` or as `n/John Doe`.
-* Items with `…`​ after them can be used multiple times including zero times e.g. `[t/TAG]...` can be used as `{nbsp}` (i.e. 0 times), `t/friend`, `t/friend t/family` etc.
+* Items with `…` after them can be used multiple times including zero times e.g. `[t/TAG]...` can be used as `{nbsp}` (i.e. 0 times), `t/friend`, `t/friend t/family` etc.
 * Parameters can be in any order e.g. if the command specifies `n/NAME p/PHONE_NUMBER`, `p/PHONE_NUMBER n/NAME` is also acceptable.
 ====
 
```

Comment:
Do we need a heading here e.g. ```List of All Commands``` before listing all the commands?

Reply:
but here don't have all the commands actually